src/pages/TasksPage.js

```diff
@@ -1,12 +1,23 @@
 import React from "react";
 import { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 
 function TasksPage() {
   const [currentTask, setCurrentTask] = useState("");
   const [taskLists, setTaskList] = useState([]);
   const [selectedTask, setSelectedTask] = useState(null);
 
+  // Get the username from localStorage
+  const username = localStorage.getItem('username');
```

Comment:
Getting `username` local storage and assigning its value to the `username` variable is not about the component, but about mounting the component. Therefore we need to use this code in `useEffect` function.